lib/std/fs/test.zig

```diff
@@ -100,6 +100,25 @@ test "create file, lock and read from multiple process at once" {
     };
 }
 
+test "open file with exclusive nonblocking lock twice (absolute paths)" {
+    const allocator = std.heap.page_allocator;
```

Comment:
I wasn't sure which allocator to use, since `std.testing.allocator` has the doccomment 
`This should only be used in temporary test programs.`
Thus, I used the page_allocator to prevent problems e.g. too many characters in cwd when using FixedBufferAllocator. Is that correct?

Reply:
`std.testing.allocator` is fine and it also helps to find missing `free`, the backing allocator has plenty of memory to run this test.

Reply:
`std.testing.allocator` is meant to be used in tests; the docs for that could probably use clarifying...